README.md

```diff
@@ -70,112 +71,41 @@ account always also needs a reference to a `User`. These dependencies are requir
 performing API calls. Take a look at [doc.bunq.com](https://doc.bunq.com) for the full
 documentation.
 
-#### Creating objects
-Creating objects through the API requires an `ApiContext`, a `requestMap` and identifiers of all
-dependencies (such as User ID required for accessing a Monetary Account). Optionally, custom headers
-can be passed to requests.
-
+The user dependecy, will always be determined for you by the SDK, for the monetary account, the sdk will use
```

Comment:
The user dependency will always be determined for you by the SDK. For the monetary account, the SDK will use your primary account (the one used for billing) if no monetary account id is provided.